firebase_remote_config/lib/src/sentry_firebase_remote_config_integration.dart

```diff
@@ -16,18 +16,26 @@ class SentryFirebaseRemoteConfigIntegration extends Integration<SentryOptions> {
 
   @override
   FutureOr<void> call(Hub hub, SentryOptions options) async {
+    options.sdk.addIntegration('SentryFirebaseRemoteConfigIntegration');
+
+    final allKeys = _firebaseRemoteConfig.getAll().keys.toSet();
+    await _updateFeatureFlags(allKeys);
```

Comment:
```suggestion
    unawaited(_updateFeatureFlags(allKeys));
```

I don't think we should await here, this can block the appRunner from executing if it takes too long